src/coreclr/tools/aot/crossgen2/Program.cs

```diff
@@ -793,7 +793,7 @@ private MethodDesc CheckAndParseSingleMethodModeArguments(CompilerTypeSystemCont
             if (method == null)
                 throw new CommandLineException(string.Format(SR.MethodNotFoundOnType, singleMethodName, singleMethodTypeName));
 
-            if (method.HasInstantiation != (singleMethodGenericArgs != null) ||
+            if (method.HasInstantiation != (singleMethodGenericArgs.Length != 0) ||
                 (method.HasInstantiation && (method.Instantiation.Length != singleMethodGenericArgs.Length)))
```

Comment:
```suggestion
            if (method.Instantiation.Length != singleMethodGenericArgs.Length)
```
I think it can be simplified to this.

Reply:
Yes, thanks!